### PR TITLE
Add autocomplete input

### LINKED ISSUE #630: Pass down autocomplete to native input
Relevant for search, but potentially relevant for other wrapped inputs ... we should pass down the native HTML autocomplete attribute - https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

### PR DESCRIPTION
**Pull request by youda97**
closes #630

### CODE REVIEW

File: src/search/search.component.ts [hunk 1]
```diff
@@ -107,6 +107,10 @@ export class Search implements ControlValueAccessor {
 	 * Sets the value attribute on the `input` element.
 	 */
 	@Input() value = "";
+	/**
+	 * Sets the autocomplete attribute on the `input` element.
+	 */
+	@Input() autocomplete = "on";
```
Comment (cal-smith) [inline src/search/search.component.ts]: We should probably type hint the possible values

```suggestion
	@Input() autocomplete: "off" | "on" = "on";
```

File: src/search/search.stories.ts [hunk 2]
```diff
@@ -11,12 +11,19 @@ storiesOf("Search", module).addDecorator(
...
+				[autocomplete]="autocomplete"
+				[disabled]="disabled"
+				[size]="size">
+			</ibm-search>
 		`,
 		props: {
 			size: select("size", ["lg", "sm"], "lg"),
 			theme: select("theme", ["dark", "light"], "dark"),
 			disabled: boolean("disabled", false),
+			autocomplete: text("autocomplete", "on"),
```
Comment (cal-smith) [inline src/search/search.stories.ts]: Maybe a `select` would make more sense here ...
```suggestion
			autocomplete: text("autocomplete", ["off", "on"], "on")
```

File: src/search/search.component.ts (hunk 1 above)
Comment (youda97) [reply src/search/search.component.ts]: There are other options though. Why are we limiting the user to just on and off?

Comment (cal-smith) [reply src/search/search.component.ts]: I totally didn't realize there were more options ... maybe instead we should link out to somewhere that lists the options - like https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#Values

Comment (youda97) [reply src/search/search.component.ts]: you want me to include this as a comment?

Comment (cal-smith) [reply src/search/search.component.ts]: Yeah, something to that effect ... just so people are aware

**Pull request merged**